docs/glossary.md

```diff
@@ -2,76 +2,78 @@
 
 # Glossary of terms
 
-In this section, you will find definition of key concepts behind the RDLS including what is a disaster, how to reduce disaster risk through disaster disaster risk management and disaster risk assessment and what are the different types of risk data needed.
+In this section, you will find definitions of key concepts behind the RDLS including what is a disaster, how to reduce disaster risk through disaster risk management and disaster risk assessment and what are the different types of risk data needed.
 
 ## Disaster risk
 
-Disaster risk is the potential loss of life, injury, or destroyed or damaged assets which could occur to a system, society or a community in a specific period of time, determined probabilistically as a function of hazard, exposure, vulnerability and coping capacity.
+Disaster risk is the potential loss of life, injury, or destroyed or damaged assets which could occur to a system, society or community in a specific period of time, determined probabilistically as a function of hazard, exposure, vulnerability and coping capacity.
 
 Disaster risk comprises different types of potential impacts and losses (direct and indirect, tangible and intangible) which are sometimes difficult to quantify. Nevertheless, with knowledge of the prevailing hazards, the patterns of population and socio-economic development and their tendency to suffer impacts, disaster risks can be assessed and mapped, at least in broad terms.
 
 Source: [UNDRR Terminology (Disaster Risk)](https://www.undrr.org/terminology/disaster-risk)
 
-## Disaster risk assessment
+## Climate and disaster risk assessment
 
-A qualitative or quantitative approach to determine the nature and extent of disaster risk by analysing potential hazards and evaluating existing conditions of exposure and vulnerability that together could harm people, property, services, livelihoods and the environment on which they depend.
+A qualitative or quantitative approach to determine the nature and extent of climate disaster risk by analysing potential hazards and evaluating existing conditions of exposure and vulnerability that together could harm people, property, services, livelihoods and the environment on which they depend.
+
+An impact or risk model (also damage model; loss model) is used to combine hazard, exposure and vulnerability components to calculate the impact or single or multiple events.
```

Comment:
```suggestion
An impact or risk model (also damage model; loss model) is used to combine hazard, exposure and vulnerability components to calculate the [impact](#disaster-impact) or single or multiple events.
```
I've not linked to the H, E or V definitions here as this definition is referring to the datasets for these components rather than the concept of them.